src/main/java/com/suchocki/parkingmeter/entity/DriverType.java

```diff
@@ -0,0 +1,60 @@
+package com.suchocki.parkingmeter.entity;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DriverType {
+	private String typeName; // typeName will be primary key in database
+
+	private List<ParkingCost> costs; // costs will store parking costs in different currencies
+
+	public DriverType() {
+		;
+	}
+
+	public DriverType(String name) {
+		this.typeName = name;
+	}
+
+	public void addCost(ParkingCost cost) {
+		if (costs == null) {
+			costs = new ArrayList<>();
+		}
+		costs.add(cost);
+	}
+
+	public String getTypeName() {
+		return typeName;
+	}
+
+	public void setTypeName(String typeName) {
+		this.typeName = typeName;
+	}
+
+	public List<ParkingCost> getCosts() {
+		return costs;
+	}
+
+	public void setCosts(List<ParkingCost> costs) {
+		this.costs = costs;
+	}
+
+	public ParkingCost getCost(Currency currency) {
+		for (ParkingCost cost : costs) {
```

Comment:
jak to by wyglądało w stylu Javy 8?

Reply:
```
return costs.stream().filter(cost -> cost.getCurrency().equals(currency)).findFirst().orElse(null);
```
I tu też lepiej byłoby zwrócić _Optionala_